keps/sig-storage/2324-volume-snapshot-namespace-transfer/README.md

```diff
@@ -0,0 +1,538 @@
+  - [Dependencies](#dependencies)
+  - [Scalability](#scalability)
+  - [Troubleshooting](#troubleshooting)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+  - [Alternative 1](#alternative-1)
+  - [Alternative 2](#alternative-2)
+- [Infrastructure Needed (Optional)](#infrastructure-needed-optional)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+## Summary
+
+Currently, VolumeSnapshot resources are tied to a specific namespace. This KEP proposes a method for transferring
+these resources between namespaces, allowing users to conveniently move these resources without manual creation
+of each independent object (VolumeSnapshot, VolumeSnapshotContent).
+
+This extends the [API for PVC Namespace Transfer](https://github.com/kubernetes/enhancements/pull/2110/files) to also apply for VolumeSnapshots.
+
+## Motivation
+
+Allow Kubernetes users to quickly and conveniently transfer VolumeSnapshots across namespaces.
+
+### Goals 
+
+- Extend the API proposed for PVC Namespace Transfer to apply for VolumeSnapshots
+
+### Non-Goals
+
+## Proposal
+
+We are going to allow users to transfer VolumeSnapshot resources between namespaces, provided
+that both the source and target namespace have a user that has agreed to the transfer.
+
+The source and target VolumeSnapshot will point to the same VolumeSnapshotContent, which
+references the backing snapshot on the storage backend.
+
+Once the transfer is completed, the source VolumeSnapshot should be deleted, so that only
+a single VolumeSnapshot stays mapped to a single VolumeSnapshotContent.
+
+### User Stories
+
+Users should be able to transfer VolumeSnapshots that reference the same VolumeSnapshotContent
+across namespaces. This allows users to quickly share snapshots, utilizing data for testing
+or backups in separate namespaces.
+
+#### Story 1
+
+The `prod` namespace contains a PVC that holds user information. VolumeSnapshots are taken of 
+this PVC on a regular basis.
+
+An existing VolumeSnapshot can be transferred to a test namespace. Once transferred, a PVC
+can be created from the snapshot for testing purposes.  
+
```

Comment:
Since I think I might be confused on the request/approval ownership thing maybe it would be good to add that detail to this user story.  In addition to "wants to xfr from prod to test" outline what actions they take to kick the process off.

Reply:
This detail has been included in the user story.